ddtrace/contrib/openai/patch.py

```diff
@@ -621,49 +643,49 @@ def handle_request(self, pin, integration, span, args, kwargs):
 def _patched_convert(openai, integration):
     def patched_convert(func, args, kwargs):
         """Patch convert captures header information in the openai response"""
-        pin = Pin._find(openai, args[0])
+        pin = Pin.get_from(openai)
         if not pin or not pin.enabled():
             return func(*args, **kwargs)
 
         span = pin.tracer.current_span()
         if not span:
             return func(*args, **kwargs)
 
-        for val in args:
```

Comment:
Why are we expecting only one response to be returned at a time?

Reply:
yes, the convert function only takes a single response, this was a bug before